### ISSUE TITLE
Unfilter Youtube

### ISSUE THREAD

**Issue by 9mido** [dtube/dtube#262]
It would be nice to have an option to not see any youtube videos and only see dtube uploaded videos only.

**Comment by techcoderx** [comment]
I agree on the filtering, or even better, remove 3rd party uploads altogether.

According to the blockchain data, 52.1% of all videos do not have any IPFS/BTFS/Skynet mirrors (the figure is 58.6% for all contents marked as original) which is insanely high. Hence, the side effect of having such filter is that much less content will be shown.

**Comment by 9mido** [comment]
Thanks for those stats. To satisfy everyone, it should be a toggle switch to filter on/off.
